src/pages/Account/Profile.tsx

```diff
@@ -60,6 +62,15 @@ const Profile: React.FunctionComponent<PropsType> = ({ location }) => {
     history.push(IS_BURN ? `/enter/step3` : nextUrl);
   };
 
+  // TODO: use functions from utils/sessionStorage or similar to retrieve the data
+  const data = sessionStorage.getItem("profileData") ?? "{}";
+  const refinedData = JSON.parse(data);
+
+  const realName =
```

Comment:
This should probably handle the case where we only have one name or the other? Could you do [firstName,lastName].filter(isTruthy).join(“ “) or similar instead?

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1667/commits/15dfaa4a6348a87a57db3386e982e12358de521c